apps/bridge-dapp/webpack.base.js

```diff
@@ -285,7 +285,7 @@ function createWebpack(env, mode = 'production') {
 
     // https://webpack.js.org/configuration/dev-server/
     devServer: {
-      port: 3000,
+      port: process.env.PORT_BRIDGE_DAPP,
```

Comment:
Consider adding a fallback port here if the env is not provided.

Reply:
i've added fallback [here](https://github.com/webb-tools/webb-dapp/pull/848/commits/c1abc8a3b4574e07cf06d2c7e34b91370b27bd16#diff-f68c8d3e5c6d2d93e76abf01b839d9699ea9946a391a42bf7d236c51f6ccd27cR288)